### ISSUE TITLE
How to increase TOTAL_MEMORY for release_performance?

### ISSUE THREAD

**Issue by hollunder** [TannerRogalsky/love.js#48]
My game seems to require that. However, I can't figure out how to actually do it. Recompile emscripten somehow? Could you give an example please?

**Comment by TannerRogalsky** [comment]
TOTAL_MEMORY is another flag that can be set on the Module object.

See [this test](https://github.com/kripken/emscripten/blob/6dc4ac5f9e4d8484e273e4dcc554f809738cedd6/tests/test_preallocated_heap_shell.html#L80) for an example of how to set it at runtime and [this configuration](https://github.com/kripken/emscripten/blob/6dc4ac5f9e4d8484e273e4dcc554f809738cedd6/src/settings.js#L57) for the default value of that setting.

**Comment by hollunder** [comment]
Thanks, the runtime example worked for me. I wouldn't know what to do with the config thing.

I added these lines to the html file in the Modules section:
        buffer: new ArrayBuffer(32*1024*1024),
        TOTAL_MEMORY: 32*1024*1024,
Except that I had to increase the memory (I simply used times 2).

Whether the ArrayBuffer line was necessary I don't know, but it seemed reasonable to include it.

The performance increase was significant compared to the compatibility. The game is now playable in Firefox as well.
